tests/PHPStan/Rules/Functions/data/fputcsv-fields-parameter.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+class Person
```

Comment:
Please don't put these test files in global namespace. Some namespace like "namespace FputcsvTest" would work.

Reply:
Added